tools/internal_ci/macos/grpc_basictests_objc.cfg

```diff
@@ -29,3 +29,8 @@ env_vars {
   key: "RUN_TESTS_FLAGS"
   value: "-f basictests macos objc dbg --internal_ci -j 1 --inner_jobs 4 --bq_result_table aggregate_results"
 }
+
+env_vars {
```

Comment:
You'd have to explicitly allow that new variable in the kokoro's job configuration (in google3) Why don't you just add an extra argument to `RUN_TEST_FLAGS` (which is already whitelisted)?